### ISSUE TITLE
Need to update for androidX version

### ISSUE THREAD

**Issue by haikalpe123** [best-flutter/flutter_swiper#55]
Just a simple request, please update this plugin so it completely compatible with a project that has been migrated to androidX. Thanks!

**Comment by vleeuwenmenno** [comment]
Seems to work here on androidX

**Comment by loolooii** [comment]
I don't think this project is compatible with AndroidX. This plugin breaks a release build when any other plugin (or the Flutter project) are using AndroidX.
